SwiftLeeds/Views/My Conference/SpeakerView.swift

```diff
@@ -40,8 +41,23 @@ struct SpeakerView: View {
                     secondaryColor: Color.primary
                 )
 
+                if presentation.videoURL != nil {
+                    CommonTileView(
+                        icon: "video.fill",
+                        primaryText: "Watch video",
+                        showChevron: true,
+                        secondaryColor: Color.primary
+                    )
+                    .accessibilityHint("Double tap to view video")
```

Comment:
Personally not keen on this kind of AX hint, there are many cases where selection is not done by "double tap". Just as valid to not have where it'll read "Watch Video. Button".

Reply:
I know this is not Apple's direction but it's how the rest of the app is so wanted consistency. Will overall them all in another PR (coming soon)

Reply:
Works for me 👍